pkg/ttl/ttlworker/job_manager.go

```diff
@@ -572,12 +572,14 @@ j:
 	}
 }
 
-func (m *JobManager) rescheduleJobs(se session.Session, now time.Time) {
+func (m *JobManager) rescheduleJobs(se session.Session, nowInSessionTz time.Time) {
+	var now time.Time
 	tz, err := se.GlobalTimeZone(m.ctx)
 	if err != nil {
 		terror.Log(err)
+		now = nowInSessionTz
 	} else {
-		now = now.In(tz)
+		now = nowInSessionTz.In(tz)
```

Comment:
Is `now` only used to determine the time window? If so, I think we can change `now` => `nowInGlobalTz`.

For `readyForLockHBTimeoutJobTables` and `lockHBTimeoutJob`, seems it should use the now with session timezone?